code/modules/reagents/reagent_containers/glass/bottle.dm

```diff
@@ -115,3 +115,9 @@
 	icon = 'icons/obj/drinks.dmi'
 	icon_state = "holyflask"
 	preloaded_reagents = list("adminordrazine" = 60)
+
+/obj/item/weapon/reagent_containers/glass/bottle/resuscitator
```

Comment:
May want to have it be a rare spawn, or blacklist it, so this valuable chem doesn't become common thanks to the loot tables.

Reply:
Sure.

Reply:
Done